dsa-solutions/lc-solutions/1400-1499/1420-build-array-where-you-can-find-the-maximum-exactly-k-comparisons.md

```diff
@@ -0,0 +1,64 @@
+---
+id: build-array-where-you-can-find-the-maximum-exactly-k-comparisons
+title: 1420. Build Array Where You Can Find The Maximum Exactly K Comparisons
+level: hard
```

Comment:
I would like to request that you please delete the line `level: hard` in this commit. This line is not part of our standard format, so removing it would help maintain consistency across all of our problem solutions. I appreciate your understanding.

Reply:
done